src/applications/vaos/tests/e2e/page-objects/DateTimeRequestPageObject.js

```diff
@@ -25,6 +25,15 @@ class DateTimeRequestPageObject extends PageObject {
   }
 
   selectFirstAvailableDate() {
+    cy.get('body').then($body => {
```

Comment:
This looks a bit odd but I haven't found a better way to do this. Essentially, I couldn't find a good way to check for the existence of an element using the selector `.vaos-calendar__calendars button[id^="date-cell"]:not([disabled]):first`. `cy.get(...)` will fail since the element doesn't exist at all and `cy.find(...)` needs a base element so I chose the base 'body' element.